### PR TITLE
Add support in setFormElement for HTML5Number element.

### PR DESCRIPTION
**Pull request by labkey-danield**
Added a handler for an HTML5 Number Field.

### CODE REVIEW

File: src/org/labkey/test/WebDriverWrapper.java [hunk 1]
```diff
@@ -3189,6 +3192,41 @@ private void setHtml5DateInput(WebElement el, Date date)
...
+        if(!valueAsNumber.equals("nan"))
+        {
+
+            int length = valueAsNumber.length();
+
+            // Double click to edit the cell.
+            Actions actions = new Actions(getDriver());
+            actions.doubleClick(el).perform();
+
+            log("WARNING: Clear did not remove the text, going to clear the field manually.");
```
Comment (labkey-tchad) [inline src/org/labkey/test/WebDriverWrapper.java]: There's no need to warn about this. We're just dealing with expected behavior. A warning implies that there is some better method that would work better.

**Comment by labkey-tchad** [comment]
Let's merge this independently of the sample management tests. This is really just bolstering the base functionality of our test API and doesn't need to be tied to those tests in perpetuity.

File: src/org/labkey/test/WebDriverWrapper.java (hunk 1 above)
Comment (labkey-danield) [reply src/org/labkey/test/WebDriverWrapper.java]: Fixed.

**Comment by labkey-danield** [comment]
Changed the title so this change can be merged with out the other SampleManagement work.

**Pull request merged**